private-templates-service/client/src/store/currentTemplate/reducers.ts

```diff
@@ -37,7 +37,7 @@ export function currentTemplateReducer(state = initialState, action: CurrentTemp
         ...state,
         templateID: "",
         templateJSON: require('../../assets/default-adaptivecards/defaultAdaptiveCard.json'),
-        templateName: "Untitled",
+        templateName: "Untitled Card",
```

Comment:
Should this be moved to the strings file? idk

Reply:
yeah